dbld/images/required-apt/artful.txt

```diff
@@ -6,6 +6,7 @@ libmongoc-dev
 librabbitmq-dev
 libsystemd-dev
 openjdk-8-jdk
+libriemann-client-dev
```

Comment:
We install this package from OBS on the following platforms:

$grep -rn libriemann-client-dev
required-obs/trusty.txt:6:libriemann-client-dev
required-obs/xenial.txt:1:libriemann-client-dev
required-obs/jessie.txt:4:libriemann-client-dev

Can we install it from the official repositories on the other platforms? Or should we change this one to?

(Note: Based on the outcome, there is a chance we can move it to the "all.txt")